src/clusterfuzz/_internal/system/environment.py

```diff
@@ -1089,6 +1089,11 @@ def is_android_cuttlefish(plt=None):
   return 'ANDROID_X86' in (plt or platform())
 
 
+def is_android_emulator(plt=None):
+  """Return True if we are on android emulator platform."""
+  return 'ANDROID_EMULATOR' in (plt or get_platform_group())
```

Comment:
I think there's a footgun in here:
```
>>> 'x' in ('y' or 'x')
False
>>> ('y' or 'x')
'y'
```
I think it works if you only want to use get_platform_group() when plt is not passed, but I don't like this kind of magic.
 Let's get rid of the plt behavior since it's not even used anywhere.

Reply:
Wow, great catch - it sounds like this should be changed to just `'ANDROID_EMULATOR' in get_platform_group()` then. Does this mean line 1094 on Android Kernel should be changed also, as well as other environments on 1084 and 1089 using `(plt or platform())`?

Reply:
To clarify my question: I think this pattern is too clever and should be gotten rid of. But I'm not sure if the original code is correct or not.

Reply:
This was indeed a convention. The intention of `plt` is for it to be an override that's considered if it's set at all. 

That said, +1 to removing that for this function since it's not used.